WordPress/Sniffs/XSS/EscapeOutputSniff.php

```diff
@@ -226,7 +226,7 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 			}
 
 			// Handle magic constants for debug functions.
-			if ( in_array( $tokens[ $i ]['code'], array( T_METHOD_C, T_FUNC_C, T_FILE, T_CLASS_C ), true ) ) {
+			if ( in_array( $tokens[ $i ]['code'], array( T_CLASS_C, T_FILE, T_FUNC_C, T_LINE, T_METHOD_C, T_NS_C, T_TRAIT_C ), true ) ) {
```

Comment:
Maybe we should also move these to a property. Make them the keys so that we can use `isset()` here instead of `in_array()`. Not sure if there would be any use for this list outside of this sniff or not; if you don't think so then just have it be a property of this sniff instead of the base sniff class.